### ISSUE TITLE
no_std support?

### ISSUE THREAD

**Issue by djmcgill** [dtolnay/proc-macro-hack#25]
What is the state of no_std support? It's my understanding that the macros all get fully expanded at compile time so if the generated code has no dependency on std then _in theory_ you could use a macro generated with this crate in a no_std environment.

Having said that, I have no idea how feasible it actually us given that `TokenStream` and `to_string` and such definitely do need std.

The error I get when trying to use this crate in a no_std environment is the expected:
```
   Compiling proc-macro-hack v0.4.0
error[E0463]: can't find crate for `std`
  |
  = note: the `thumbv6m-none-eabi` target may not be installed
```

**Comment by dtolnay** [comment]
Thanks, I released 0.4.1 with the line that should make this work.

**Comment by djmcgill** [comment]
Thanks for the quick response! If you would like to see how we're using it the proof of concept I put together is here: https://github.com/djmcgill/atsamd21-rs/blob/mashup-iden-test/arduino_mkrzero/src/lib.rs
